tests/end2end/cypress/integration/zoom-to-layer-ghaction.js

```diff
@@ -0,0 +1,60 @@
+describe('Zoom to layer', function() {
+    it('Projection EPSG:4326', function(){
+        // Go to world 4326
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=world-4326')
+
+        // Zoom to layer rectangle
+        cy.get('#layer-rectangle > td:first > span.label').click()
+        cy.get('#sub-dock button.layerActionZoom').click()
+
+        // Click on the map to get a popup
+        cy.get('#map').click(480,340)
```

Comment:
Does `cy.mapClick` would help to maintain coordinates ?
I think it would be nice to make it uniform.

(and others references)

Reply:
Yes